winres/winres.json

```diff
@@ -0,0 +1,54 @@
+{
+  "RT_GROUP_ICON": {
+    "OTHER": {
+      "0000": "hazelcast-icon-flat-sm.png"
+    }
+  },
+  "RT_MANIFEST": {
+    "#1": {
+      "0409": {
+        "identity": {
+          "name": "",
+          "version": ""
+        },
+        "description": "",
+        "minimum-os": "win7",
+        "execution-level": "as invoker",
+        "ui-access": false,
+        "auto-elevate": false,
+        "dpi-awareness": "system",
+        "disable-theming": false,
+        "disable-window-filtering": false,
+        "high-resolution-scrolling-aware": false,
+        "ultra-high-resolution-scrolling-aware": false,
+        "long-path-aware": false,
+        "printer-driver-isolation": false,
+        "gdi-scaling": false,
+        "segment-heap": false,
+        "use-common-controls-v6": false
+      }
+    }
+  },
+  "RT_VERSION": {
+    "#1": {
+      "0409": {
+        "info": {
+          "0409": {
+            "FileDescription": "Command line client for Hazelcast",
+            "InternalName": "hzc",
+            "LegalCopyright": "©Hazelcast, Inc.",
+            "OriginalFilename": "hzc.exe",
+            "ProductName": "hzc",
+            "Comments": "",
+            "CompanyName": "Hazelcast, Inc.",
+            "FileVersion": "",
+            "LegalTrademarks": "",
+            "PrivateBuild": "",
+            "ProductVersion": "",
+            "SpecialBuild": ""
```

Comment:
What's the difference between `FileVersion` and `ProductVersion`? Should we put the CLC version in one/both of them?

Reply:
I removed them since we pass them via command flag to "go-winres"

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/118/commits/a5e0fcc2528d594f911eada5befecc6539550643